cmd/spire-agent/cli/command/run.go

```diff
@@ -27,6 +28,8 @@ const (
 	defaultDataDir   = "."
 	defaultLogLevel  = "INFO"
 	defaultPluginDir = "conf/plugin/agent"
+
+	defaultUmask = -1
```

Comment:
Is `-1` a legal umask value? We should set the default to be as conservative as possible

Reply:
`-1` means umask must not be changed. That's the default behavior.

Reply:
The default should be 077 IMO - it should be something that we know is "secure"